kfs-core/src/main/java/edu/arizona/kfs/fp/document/authorization/ProcurementCardAccountingLineAuthorizer.java

```diff
@@ -0,0 +1,22 @@
+package edu.arizona.kfs.fp.document.authorization;
+
+import java.util.Map;
+
+import org.kuali.kfs.sys.businessobject.AccountingLine;
+import org.kuali.kfs.sys.document.AccountingDocument;
+import org.kuali.rice.kim.api.identity.Person;
+
+public class ProcurementCardAccountingLineAuthorizer extends org.kuali.kfs.fp.document.authorization.ProcurementCardAccountingLineAuthorizer {
+    
+    /**
```

Comment:
A two * comment will get picked up by the javadoc processor, but this method-level comment does not list either the args or the return type. Please either fully flesh out as javadoc, or alternatively delete one of the stars so it doesn't make it into docs with no signature explanation.